### ISSUE TITLE
[FEATURE REQUEST] New function: capitalize()

### ISSUE THREAD

**Issue by silvioprog** [brazilian-utils/brazilian-utils#99]
Hi.

It would be nice to provide some function like this:

```js
capitalize("esponja de aço 60g") // Capitalizes to "Esponja de Aço 60G"
capitalize("fulano de tal") // Capitalizes to "Fulano de Tal"
```

cheers

**Comment by silvioprog** [comment]
Awesome! ^^

I'll try it and send as new PR ASAP ...

**Comment by silvioprog** [comment]
Sent as PR https://github.com/brazilian-utils/brazilian-utils/pull/100.

**Comment by hyanmandian** [comment]
Closed by #101
